### DISCUSSION TITLE
No-Go Area Roborock V1 (Carpet)

### DISCUSSION THREAD

**Discussion by elgatobavaria** [Hypfer/Valetudo#1741] [Q&A / Support] [answered]
Hi guys,

i try to use the No-Go-Area funciton from Valetudo but i dont know how to do this and cant find any information about. Would be glad if anybody can help me. I have one carpet in my apartment and it should not be allowed to the robot to pass this one.

I already saw the hint to use zones to avoid cleaning the carpet. But i dont want the cleaner to pass the carpet.

Vaccum: Xiaomi Roborock V1
Valetudo Version: 23.02.0

Best regards Mathias

**Comment by Hypfer** [comment, marked as answer]
The V1 firmware does not support virtual restrictions. There's no way around that unfortunately
